.github/workflows/pr.yml

```diff
@@ -122,6 +122,35 @@ jobs:
         run: dotnet run integration test TestProvider
       - name: TestDeletedWith
         run: dotnet run integration test TestDeletedWith
+
+  info:
+    name: gather
+    runs-on: ubuntu-latest
+    outputs:
+      version: "${{ steps.version.outputs.version }}"
+    steps:
+      - uses: actions/checkout@v4
+      - name: Git describe
+        id: ghd
+        uses: proudust/gh-describe@v1
+      - name: Stitch version together
+        id: version
+        run: |
+          TAG="${{ steps.ghd.outputs.tag }}"
```

Comment:
Do we not want to make use of the current version info like the normal release does? 
https://github.com/pulumi/pulumi-dotnet/blob/main/.github/workflows/release-dotnet-provider.yml#L34C1-L38C23

I'd expect this to query changie for the _next_ version and then add a dev hash to the end of that.

Reply:
Yeah good point, I've changed it to using changie.